eegnb/experiments/visual_ssvep/ssvep.py

```diff
@@ -76,25 +41,7 @@ def get_possible_ssvep_freqs(frame_rate, stim_type="single"):
         return freqs
 
     def init_flicker_stim(frame_rate, cycle, soa):
-        """Initialize flickering stimulus.
-        Get parameters for a flickering stimulus, based on the screen refresh
-        rate and the desired stimulation cycle.
-        Args:
-            frame_rate (float): screen frame rate, in Hz
-            cycle (tuple or int): if tuple (on, off), represents the number of
-                'on' periods and 'off' periods in one flickering cycle. This
-                supposes a "single graphic" stimulus, where the displayed object
-                appears and disappears in the background.
-                If int, represents the number of total periods in one cycle.
-                This supposes a "pattern reversal" stimulus, where the
-                displayed object appears and is replaced by its opposite.
-            soa (float): stimulus duration, in s
-        Returns:
-            (dict): dictionary with keys
-                'cycle' -> tuple of (on, off) periods in a cycle
-                'freq' -> stimulus frequency
-                'n_cycles' -> number of cycles in one stimulus trial
-        """
```

Comment:
Why remove this?

Reply:
Ignore the visual_ssvep for now, still doing it. But I'm gonna keep that in the load_stimulus or present_stimulus function